skimage/transform/radon_transform.py

```diff
@@ -216,26 +216,37 @@ def iradon(radon_image, theta=None, output_size=None,
     img = np.pad(radon_image, pad_width, mode='constant', constant_values=0)
 
     # Construct the Fourier filter
-    f = fftfreq(projection_size_padded).reshape(-1, 1)   # digital frequency
-    omega = 2 * np.pi * f                                # angular frequency
-    fourier_filter = 2 * np.abs(f)                       # ramp filter
+    n1 = np.arange(0, projection_size_padded / 2 + 1, dtype=np.int)
+    n2 = np.arange(projection_size_padded / 2 - 1, 0, -1, dtype=np.int)
```

Comment:
Floor divide here as well.
```suggestion
    n2 = np.arange(projection_size_padded // 2 - 1, 0, -1, dtype=np.int)
```